traffic_ops/traffic_ops_golang/crconfig/edgelocations.go

```diff
@@ -49,14 +49,49 @@ and (st.name = 'REPORTED' or st.name = 'ONLINE' or st.name = 'ADMIN_DOWN')
 
 	for rows.Next() {
 		cachegroup := ""
+		primaryCacheID := 0
 		ttype := ""
+		var fallbackToClosest *bool
 		latlon := tc.CRConfigLatitudeLongitude{}
-		if err := rows.Scan(&cachegroup, &ttype, &latlon.Lat, &latlon.Lon); err != nil {
+		if err := rows.Scan(&cachegroup, &primaryCacheID, &ttype, &latlon.Lat, &latlon.Lon, &fallbackToClosest); err != nil {
 			return nil, nil, errors.New("Error scanning cachegroup: " + err.Error())
 		}
 		if ttype == RouterTypeName {
 			routerLocs[cachegroup] = latlon
 		} else {
+			q := `select cachegroup.name from cachegroup_fallbacks
+join cachegroup on cachegroup_fallbacks.backup_cg = cachegroup.id
+and cachegroup_fallbacks.primary_cg = $1
```

Comment:
I missed this in my previous review, but this query is missing `order by cachegroup_fallbacks.set_order`. Without this the `set_order` ordering is not enforced.

Reply:
Right. I missed that. Addressed.